### PR TITLE
add python script for lesson contributor count

### PR DESCRIPTION
**Pull request by maneesha**
See conversation in https://github.com/carpentries/carpentries.org/issues/449

### CODE REVIEW

**Comment by maneesha** [comment]
Do not merge - this builds successfully but the python code itself broke somewhere along the way returning incorrect counts.

File: python/lesson_contributor_count.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import requests
 import json
 
-token = "GITHUB_TOKEN"
+token = GITHUB_TOKEN
```
Comment (fmichonneau) [inline python/lesson_contributor_count.py]: I _think_ you need:

```python
import os
token = os.environ["GITHUB_TOKEN"]
```

Comment (maneesha) [reply python/lesson_contributor_count.py]: Yup, I saw that after I tried pushing before.  Another build attempt is happening now. :-)

I thought that the existing `GITHUB_PAT` was invalid which is why I tried adding my own token.  If the existing `GITHUB_PAT` is valid, is it OK to keep just that one?  I believe it's tied to your account so I wanted to make sure first.

Comment (fmichonneau) [reply python/lesson_contributor_count.py]: Yes, you can use the existing GITHUB_PAT and delete the other one on Travis.

**Comment by maneesha** [comment]
@fmichonneau  Do you know why a build isn't being triggered for the latest two commits in this PR?

**Comment by maneesha** [comment]
Thanks for catching that.  

It builds now  but  [cat lesson_contributor_count.json](https://travis-ci.com/carpentries/amy-feeds/builds/129896224#L1041) returns an empty file.  It does work locally, populating the `json` file with the Python dict.  

I'm going to merge this and see if it works when it actually deploys (since it doesn't seem like it will break anything else).

**Comment by maneesha** [comment]
Thanks for catching that.  

It builds now  but  [cat lesson_contributor_count.json](https://travis-ci.com/carpentries/amy-feeds/builds/129896224#L1041) returns an empty file.  It does work locally, populating the `json` file with the Python dict as expected.  

I'm going to merge this to see if it works on deploy (as it does not seem to break anything else)

**Pull request merged**